data/card_catalogue.json

```diff
@@ -413,33 +475,23 @@
         {
           "title": "Ender Pearl",
           "upgrade": "",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
-            "rules": "Teleport to a space up to 10 tiles away and take 2 damage",
+            "rules": "Teleport to any visible space. Take damage equal to the number of squares moved.",
```

Comment:
Not sure if the numbers it had before were a critical minecraft reference or something. I think this is a fun gambit mechanic

Reply:
I like it. IMO we should strip the deck of all non-cardmaster references at some point.

Reply:
Sure, I can re-flavour this. Not sure what we're going to do about the popular proplauncher 🤔

Reply:
I've added a ticket to discuss removing all references